internal/core/items/component_container.rs

```diff
@@ -219,10 +220,21 @@ impl Item for ComponentContainer {
 
     fn render(
         self: Pin<&Self>,
-        _backend: &mut super::ItemRendererRef,
-        _item_rc: &ItemRc,
-        _size: LogicalSize,
+        backend: &mut super::ItemRendererRef,
+        item_rc: &ItemRc,
+        size: LogicalSize,
     ) -> RenderingResult {
+        if let Some(item_tree) = self.item_tree.borrow().clone() {
+            let item_tree = vtable::VRc::borrow_pin(&item_tree);
+            let root_item = item_tree.as_ref().get_item_ref(0);
+            let window_item =
+                crate::items::ItemRef::downcast_pin::<crate::items::WindowItem>(root_item).unwrap();
+
+            let mut rect = Rectangle::default();
+            rect.background.set(window_item.background());
+            backend.draw_rectangle(std::pin::pin!(rect).as_ref(), item_rc, size);
```

Comment:
To fix no_std build:

```suggestion
            backend.draw_rectangle(core::pin::pin!(rect).as_ref(), item_rc, size);
```